[#965] Changelog update for the fixes (#966)
Electric-Coin-Company/zashi-ios: CHANGELOG.md

### Changed
- The background of Onboarding and some of the Settings screen has been updated to show a subtle texture of a grid pattern.

> - The sapling address + QR is present on a Receive tab for the testnet only.

Reviewer: ```suggestion
- The sapling address + QR code has been restored on the Receive tab (for testnet only.)
```

### Fixed

> - The sending got stuck because the alert was missing. When the send request fails, the alert is presented and the spinner is stopped and hidden.

Reviewer: ```suggestion
- Fixed a bug that caused spends to appear to be stuck.
```

> - The empty message on a confirmation screen has been removed. Rendered only when there is some message actually written. 

Reviewer: ```suggestion
- The confirmation screen has been altered such that the message bubble is rendered only when the message is non-empty. 
```
